### PR TITLE
changes to use pysecuritas api

### PR DESCRIPTION
**Pull request by nragon**

### CODE REVIEW

File: custom_components/securitas_direct/manifest.json [hunk 1]
```diff
@@ -1,12 +1,10 @@
 {
   "domain": "securitas_direct",
   "name": "Securitas Direct",
-  "documentation": "https://github.com/segalion/securitasdirect",
-  "issue_tracker": "https://github.com/segalion/securitasdirect/issues",
+  "documentation": "https://www.home-assistant.io/integrations/securitas_direct",
```
Comment (jmservera) [inline custom_components/securitas_direct/manifest.json:4]: You are replacing the original documentation with a broken link, I think you need to undo this change (and issue_tracker as well).

Comment (nragon) [reply custom_components/securitas_direct/manifest.json:4]: I'm following HA guidelines (https://developers.home-assistant.io/docs/creating_integration_manifest) in order to submit the integration.

Comment (jmservera) [reply custom_components/securitas_direct/manifest.json:4]: Ok. Sorry, I didn't see the issue #23 
BTW, I noticed that after changing to your code, now the HA component appears as "unnamed device" and has no unique id, do you know how to fix it?

Comment (nragon) [reply custom_components/securitas_direct/manifest.json:4]: It's probably due the fact I've removed this
`'alias': hub.session.alias` from `device_state_attributes`

Alias comes from an operation ("INS"). I'll try to add that at the beginning of the connection and merge it here asaic.

Comment (jmservera) [reply custom_components/securitas_direct/manifest.json:4]: Yes! This last commit did fix it. Awesome. Thanks!
